src/commands/importDocuments.ts

```diff
@@ -57,13 +57,17 @@ export async function importDocuments(uris: vscode.Uri[] | undefined, collection
 }
 
 async function askForDocuments(): Promise<vscode.Uri[]> {
+    let rootPath = vscode.workspace.rootPath;
+    if (!rootPath) {
+        throw new Error("No workspace found. Please open a workspace");
```

Comment:
I don't think this should block the user from importing documents. I'm pretty sure `defaultUri` is an optional parameter below or you could consider doing `os.homedir()` if it's not optional

Reply:
Sounds good.